xen/drivers/passthrough/device_tree.c

```diff
@@ -187,7 +187,7 @@ int iommu_do_dt_domctl(struct xen_domctl *domctl, struct domain *d,
         ret = iommu_deassign_dt_device(d, dev);
 
         if ( ret )
-            printk(XENLOG_G_ERR "XEN_DOMCTL_assign_dt_device: assign \"%s\""
+            printk(XENLOG_G_ERR "XEN_DOMCTL_deassign_dt_device: assign \"%s\""
                    " to dom%u failed (%d)\n",
```

Comment:
As a said in pull request description I have pushed all patches I have, but not of all are mine.
I did this just for proving ability for everyone to compile and test out of the box)
My patches are up from 4ff5c17 xen/device-tree: Add dt_count_phandle_with_args helper.